test/e2e_node/runner/remote/run_remote.go

```diff
@@ -359,7 +361,6 @@ func prepareGceImages() (*internalImageConfig, error) {
 				metadata:        getImageMetadata(metadata),
 				kernelArguments: imageConfig.KernelArguments,
 				machine:         imageConfig.Machine,
-				tests:           imageConfig.Tests,
```

Comment:
thank you for removing this. Did you check we removed all the instances of this?

Reply:
Yes, the only reference I could find is https://github.com/kubernetes/test-infra/pull/26017 but it was deleted in that PR.